servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/ReservableRequestConcurrencyControllers.java

```diff
@@ -113,6 +113,7 @@ public String toString() {
 
     private abstract static class AbstractReservableRequestConcurrencyController
             implements ReservableRequestConcurrencyController {
+        private static final HttpEventKey<ConsumableEvent<Integer>> MAX_CONCURRENCY_KEY = MAX_CONCURRENCY_NO_OFFLOADING;
```

Comment:
Do we anticipate this changing in the future or something? I would have personally just inlined it.

Reply:
No, just wanted to make sure we use the same key everywhere: as an argument for `transportEvents` and for logging. 
It helps to make it clear, because we also have a public `HttpEventKey.MAX_CONCURRENCY` key that offloads events.

Reply:
I did see the `MAX_CONCURRENCY` which is part of why this seemed somewhat less clear, but it's obviously subjective. Feel free to ignore, especially if you don't have anything else to commit with it.